commerce-frontend-js/dev/public/gallery.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+
+	<meta
+		content="width=device-width, initial-scale=1, shrink-to-fit=no"
+		name="viewport"
+	/>
+	<!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css" />
+	-->
```

Comment:
can you remove this comment? AFAIU, it doesn't add any value here (talked this over with Fabio recently on a similar PR -- feel free to ping me or him if you disagree)